docs/install.sh

```
@@ -0,0 +1,238 @@
+    VERSION="$3"
+    INSTALL_DIR="$4"
+    
+    BINARY_NAME_FULL="${BINARY_NAME}-${VERSION}-${OS}-${ARCH}"
+    DOWNLOAD_URL="https://github.com/${REPO_OWNER}/${REPO_NAME}/releases/download/${VERSION}/${BINARY_NAME_FULL}"
+    
+    info "Downloading ${BINARY_NAME} ${VERSION} for ${OS}/${ARCH}..."
+    info "URL: ${DOWNLOAD_URL}"
+    
+    # Create temp directory
+    TMP_DIR=$(mktemp -d)
+    trap "rm -rf ${TMP_DIR}" EXIT
```

Shellcheck (0.10.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)